x/resource/client/cli/tx_create_resource.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"io/ioutil"
+
+	cheqdcli "github.com/cheqd/cheqd-node/x/cheqd/client/cli"
+	"github.com/cheqd/cheqd-node/x/resource/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+)
+
+func CmdCreateResource() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-resource [collection-id] [id] [name] [resource-type] [file-path] [ver-method-id-1] [priv-key-1] [ver-method-id-N] [priv-key-N] ...",
```

Comment:
Maybe use `--file` or similar to denote the file path? That allows the file path to be in any order

Reply:
Disambiguate required parameters by checking for flag rather than expecting a specific order: `create-resource --collection-id [collection-id] --resource-id [id] --resource-name [name] --resource-type [resource-type] --resource-file [file-path] [ver-method-id-1] [priv-key-1] [ver-method-id-N] [priv-key-N] ...`